docs/markdown/Getting Started/getting-started/installation.md

```diff
@@ -9,7 +9,7 @@ updatedAt: "2022-07-12T00:00:00.000Z"
 You can download an installer script that will install the Pants binary with this command:
```

Comment:
We should change this sentence to "You can download and run an installer script..."

Reply:
So let's just fix this, and then I'll kick off a CI run. Thanks!

Reply:
Like this?
```suggestion
You can download and run an installer script that will install the Pants binary with this command:
```

Reply:
Exactly